frontend/src/app/(routes)/governance/DepositPopup.tsx

```diff
@@ -0,0 +1,80 @@
+'use client';
```

Comment:
Where can I see UI for this?

Reply:
I haven't displayed anywhere, because there is no data for it. I had created the UI for it